PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties

```diff
@@ -2,7 +2,7 @@ javaVersion=11
 org.gradle.jvmargs='-Duser.language=en'
 sources=false
 intellij_version=IC-LATEST-EAP-SNAPSHOT
-dep_plugins=org.intellij.scala:2020.3.8
+dep_plugins=org.intellij.scala:2021.1.10
 applicationinsights.key=57cc111a-36a8-44b3-b044-25d293b8b77c
 updateVersionRange=true
-patchPluginXmlSinceBuild=203.5419.21
+patchPluginXmlSinceBuild=211.6085.26
```

Comment:
New EAP version is released. We should do the following version upgrade.
1. Upgrade Intellij scala plugin version to `2021.1.12` based on [plugin version description](https://plugins.jetbrains.com/plugin/1347-scala/versions/eap)
2. Upgrade lower bound of IntelliJ version to `211.6432.7` based on [official site description](https://www.jetbrains.com/idea/nextversion/#section=windows)